### PR TITLE
fix: floating dropdown position

### PR DESCRIPTION
**Pull request by rustem-nasyrov**
Close: #2250

## Description
Fix dropdown position is broken after SPA route change in nuxt apps.

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/composables/useDropdown.ts [hunk 1]
```diff
@@ -169,6 +151,25 @@ export const useDropdown = (
   }
 
   watchPostEffect(() => {
+    const rootRef = computed(() => {
```
Comment (m0ksem) [inline packages/ui/src/composables/useDropdown.ts:154]: Computed must be created in setup function, not in sub function. This cause creating new computed each time anchor or content DOM rect change.

**Review by m0ksem — CHANGES_REQUESTED** [review]
I think the problem here is that root passed from VaDropdown is not reactive ref because of useTemplateRef.

File: packages/ui/src/composables/useDropdown.ts (hunk 1 above)
Comment (rustem-nasyrov) [reply packages/ui/src/composables/useDropdown.ts:154]: What if replace it with a ref?

**Comment by rustem-nasyrov** [comment]
External bug, reproduces only in the nuxt. Here is a link to the bug: https://github.com/nuxt/framework/issues/6845

**Review by m0ksem — APPROVED** [review]

**Pull request merged**